### PR TITLE
winuwp: Add multi-touch support

### PR DESCRIPTION
**Pull request by jnschulze**
Now that multi touch is supported for Win32, it's time to implement it for UWP as well. This PR does so.

Fixes https://github.com/flutter/flutter/issues/70201

Linked issues:
- https://github.com/flutter/flutter/issues/70201
- #27863 

Tested with flutter_gallery.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by clarkezone** [comment]
Is there a test app i can try out to see this working?

**Comment by jnschulze** [comment]
> Is there a test app i can try out to see this working?

The `flutter_gallery` has an example called `2D transformations`. This at least allows you to some scaling using a pinch gesture.

**Comment by clarkezone** [comment]
I will take a look in the next day or two (sorry for delay)

**Comment by clarkezone** [comment]
I'm blocked currently due to inability to build the UWP target on my machine.

File: shell/platform/windows/flutter_window_winuwp.cc [hunk 1]
```diff
@@ -11,10 +11,11 @@ namespace flutter {
...
 
-// TODO(clarkezone): Determine pointer ID in
-// OnPointerPressed/OnPointerReleased/OnPointerMoved in order to support multi
-// touch. See https://github.com/flutter/flutter/issues/70201
-static constexpr int32_t kDefaultPointerDeviceId = 0;
+// Minimum pointer ID that gets emitted by the pointer ID generator.
+static constexpr auto kMinPointerId = 0;
+
+// Maximum pointer ID that gets emitted by the pointer ID generator.
+static constexpr auto kMaxPointerId = 128;
```
Comment (cbracken) [inline shell/platform/windows/flutter_window_winuwp.cc:18]: Use the explicit type for top-level constant declarations -- in this case, `uint32_t`, for pointer IDs. 

We're not particularly consistent about this ourselves, but for the most part, try to restrict use of `auto` to locals where the type is evident from the right hand side..

**Comment by cbracken** [comment]
Overall this LGTM; @clarkezone any thoughts on ways we could test this?

**Review by cbracken — COMMENTED** [review]
LGTM with one nit, plus would like to get @clarkezone's thoughts on testing.

**Comment by clarkezone** [comment]
Please can you update this to top of tree?  I'm onblocked and standing by to test

**Comment by jnschulze** [comment]
> Please can you update this to top of tree? I'm onblocked and standing by to test

done.

**Review by clarkezone — APPROVED** [review]
LGTM

**Pull request merged**